modules/stats.php

```diff
@@ -893,13 +893,6 @@ function stats_dashboard_head() { ?>
 /* <![CDATA[ */
 jQuery(window).load( function() {
 	jQuery( function($) {
```

Comment:
My assumption is that it was needed to actually wait for the load event, I will try to get rid of it and see if it helps.